### ISSUE TITLE
doesn't launch whatsoever

### ISSUE THREAD

**Issue by marciebeans** [huderlem/porymap#431]
I have downloaded fresh precompiled copies multiple times, and have cleared cfg data per [this post,](https://github.com/huderlem/porymap/issues/183) but despite this, absolutely nothing happens when I open up the exe file. No error message, even, just nothing.

**Comment by garakmon** [comment]
Operating system?

**Comment by marciebeans** [comment]
windows 7

**Comment by garakmon** [comment]
Have you checked the log file? Can you paste some of the last lines here please.

**Comment by marciebeans** [comment]
The only error in the log is from 2020, from a completely different version than i'm trying to run. I could be wrong but my my guess is that wouldn't have any helpful info for you here...

**Comment by garakmon** [comment]
Okay, could be that Qt6 does not support Windows 7 and the latest version was compiled with Qt6

**Comment by marciebeans** [comment]
Bugger. 4.4 was doing this to me too, so do you know the latest non-Qt6 version then? Or do you recommend I compile it myself?

**Comment by garakmon** [comment]
If you can compile yourself that would be good, porymap is still compatible with Qt >= 5.14. 
Based on the changelog I would have expected 4.4.0 to have been built with Qt5, but Qt6 was released a couple of weeks before 4.4.0 so it's possible it was not. 4.3.1 is definitely Qt5.

**Comment by marciebeans** [comment]
Okay, I'll just compile it myself to not miss bugfixes/features/etc. Thank you for being so helpful!

**Comment by marciebeans** [comment]
Welp, after some finagling with QT and _that also_ refusing to work for me, I tried 4.3 and that didn't run either. At a loss for what the reason here could be.

**Comment by garakmon** [comment]
what is the most recent version that will open for you?

**Comment by marciebeans** [comment]
I hadn't tried them all, so since you asked, I tried 1.0, and even that didn't work. Might not be a Qt6 problem, I guess, but... I don't know where to go from here.

**Comment by garakmon** [comment]
You deleted the config file in the appdata folder?

**Comment by marciebeans** [comment]
yeah, i deleted everything in appdata/pret to see if it would do anything

**Comment by garakmon** [comment]
porymap has worked on this machine before?

**Comment by marciebeans** [comment]
It did at some point, according to those logs, but I really don't remember it tbh.

**Comment by GriffinRichards** [comment]
Closing stale question. Feel free to follow-up if you're still having problems.
